api/app/auto_spatial_advisory/hfi_minimum_wind_speed.py

```diff
@@ -77,6 +77,7 @@ async def process_min_wind_speed_by_zone(session: AsyncSession, run_parameters_i
     fire_zone_shape_type_id = await get_fire_zone_unit_shape_type_id(session)
     zone_units = await get_fire_zone_units(session, fire_zone_shape_type_id)
     srid = await get_table_srid(session, Shape) or 3005  # default to bc albers
+    advisory_id_lut = await get_hfi_threshold_ids(session)
```

Comment:
`lut`? look up threshold?

Reply:
It's a term for lookup table but I can expand if it helps: https://en.wikipedia.org/wiki/Lookup_table